plugins/outputs/influxdb_v2/http.go

```diff
@@ -402,11 +404,13 @@ func (c *httpClient) addHeaders(req *http.Request) {
 	}
 }
 
-func makeWriteURL(loc url.URL, org, bucket string) (string, error) {
-	params := url.Values{}
+func makeWriteURL(loc url.URL, params url.Values, bucket string) string {
 	params.Set("bucket", bucket)
-	params.Set("org", org)
+	loc.RawQuery = params.Encode()
+	return loc.String()
+}
 
+func prepWriteURL(loc url.URL, org string) (*url.URL, url.Values, error) {
```

Comment:
Can we please use a full name instead of shortened ones? Maybe
```suggestion
func prepareURL(loc url.URL, org string) (*url.URL, url.Values, error) {
```

I also wonder if we shouldn't copy this over into the caller location instead of making this a one-time-called function...